build/nuke/Build.cs

```diff
@@ -1,13 +1,20 @@
 using System.IO;
 using System.Linq;
 using Nuke.Common;
+using Nuke.Common.CI.GitHubActions;
 using Nuke.Common.IO;
 using Nuke.Common.Tools.MSBuild;
 using Nuke.Common.Utilities.Collections;
 using static Nuke.Common.EnvironmentInfo;
 using static Nuke.Common.IO.FileSystemTasks;
 
-//[CheckBuildProjectConfigurations]
+[GitHubActions("ci",
```

Comment:
Does this generate the CI yml on its own?

Reply:
yep, the name there matches with .yml file. Executing dotnet nuke tool is generating updates.

Reply:
Dope :)